Assets/VRTK/Scripts/Interactions/VRTK_InteractTouch.cs

```diff
@@ -33,9 +33,13 @@ public struct ObjectInteractEventArgs
     /// <example>
     /// `VRTK/Examples/005_Controller/BasicObjectGrabbing` demonstrates the highlighting of objects that have the `VRTK_InteractableObject` script added to them to show the ability to highlight interactable objects when they are touched by the controllers.
     /// </example>
-    [RequireComponent(typeof(VRTK_ControllerActions))]
     public class VRTK_InteractTouch : MonoBehaviour
     {
+        [Tooltip("Will look for local instance is left null.")]
```

Comment:
This should probably also explain the field's usage and explain more explicitly that `gameObject` is searched. Something along the lines of "The controller events instance to use. If this is left empty the game object this script is attached to will be searched for an instance of `VRTK_ControllerEvents`.